<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>custom_signal::SignalConfigDialog</class>
 <widget class="QDialog" name="custom_signal::SignalConfigDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>178</width>
    <height>204</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sound Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>6</number>
   </property>
   <property name="topMargin">
    <number>6</number>
   </property>
   <property name="rightMargin">
    <number>6</number>
   </property>
   <property name="bottomMargin">
    <number>6</number>
   </property>
   <item>
    <widget class="QGroupBox" name="sounds_group">
     <property name="title">
      <string>Sound</string>
     </property>
     <layout class="QVBoxLayout" name="sounds_layout">
      <item>
       <layout class="QHBoxLayout" name="defaults_layout" stretch="0,1">
        <item>
         <widget class="QRadioButton" name="default_selected">
          <property name="text">
           <string notr="true"/>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="defaults_box"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="custom_layout">
        <item>
         <widget class="QRadioButton" name="custom_selected">
          <property name="text">
           <string>custom</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="custom_browse_btn">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string notr="true">...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="volume_group">
     <property name="title">
      <string>Volume</string>
     </property>
     <layout class="QHBoxLayout" name="volume_layout">
      <item>
       <widget class="QSlider" name="volume_slider">
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="value">
         <number>40</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="preview_btn">
        <property name="text">
         <string notr="true">...</string>
        </property>
        <property name="icon">
         <iconset resource="../custom_signal.qrc">
          <normaloff>:/custom_signal/play.svg</normaloff>:/custom_signal/play.svg</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../custom_signal.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>custom_signal::SignalConfigDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>174</x>
     <y>177</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>193</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>custom_signal::SignalConfigDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>174</x>
     <y>183</y>
    </hint>
    <hint type="destinationlabel">
     <x>183</x>
     <y>193</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>default_selected</sender>
   <signal>toggled(bool)</signal>
   <receiver>defaults_box</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>25</x>
     <y>48</y>
    </hint>
    <hint type="destinationlabel">
     <x>67</x>
     <y>46</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>custom_selected</sender>
   <signal>toggled(bool)</signal>
   <receiver>custom_browse_btn</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>64</x>
     <y>71</y>
    </hint>
    <hint type="destinationlabel">
     <x>150</x>
     <y>74</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
